net/mwan3/files/etc/init.d/mwan3

```diff
@@ -1,9 +1,8 @@
 #!/bin/sh /etc/rc.common
 
-. /lib/functions.sh
-. /lib/mwan3/common.sh
-. /lib/functions/network.sh
-. /lib/mwan3/mwan3.sh
+. $IPKG_INSTROOT/lib/mwan3/common.sh
```

Comment:
Could you please add quotation marks around this?

Reply:
It seems that the practice in openwrt is to not quote those (eg `/etc/rc.common`).

Reply:
I would say this is not uniform.

package feed:
https://github.com/openwrt/packages/blob/ffdbc7a637df8e3f39dd6916405f78840edccfb4/net/ocserv/files/ocserv.init#L6
https://github.com/openwrt/packages/blob/0038d6e971bf98ef440e15cc3a71e643d4fa65ba/net/nut/files/nut-monitor.init#L174
https://github.com/openwrt/packages/blob/c05e92a58d84c20750f2974da6a89e872d1a1041/net/fwknop/files/fwknopd.init#L11

openwrt:
https://github.com/openwrt/openwrt/blob/5792f6a1043d686d4ee3c8b8f7ce070ac1bb3e5d/package/system/procd/files/procd.sh#L43
https://github.com/openwrt/openwrt/blob/0232bf601d6c8c174faa4c5bac1f6c86b20853c9/package/base-files/files/etc/rc.common#L4

Reply:
Most init scripts move the include to the function so that they don't get an error on install during image build.
That would be one possibility, but I don't like that.
If i need it in different functions in init i always have to include it.

For example https://github.com/openwrt/packages/blob/be6312296c6132d494505f5c6f0b7bb4bb3aec6d/utils/collectd/files/collectd.init#L191
https://github.com/openwrt/packages/blob/10da8b0b8e45136cb5b10c1fd105ea0844bd998e/net/banip/files/banip.init#L94

Reply:
Okay, looks like it can go either way. I'll add the quotation marks around it to accommodate shellcheck.